### DISCUSSION TITLE
Selected content have not appended to the active file.

### DISCUSSION THREAD

**Discussion by theotheo** [coddingtonbear/obsidian-web#165] [Bug Report]
### Describe the bug

Selected content have not appended to the active file. 

I'm not sure where the problem. It is possible that Obsidian itself ignore file updates. Because it seems that the active file was changed.

[obsidian-local-rest-api.bug_report.json](https://github.com/coddingtonbear/obsidian-web/files/14233492/obsidian-local-rest-api.bug_report.json)

### To Reproduce

1. Click to 'Obsidian Web' icon 
2. Click on 'Send to Obsidian' button
3. Notice "All Done!" message
4. Open Obsidian window
5. Doesn't see any changes
6. Disappoint 

### Operating System

Ubuntu 20.04

### Browser

Chrome

### Browser Version

Version 119.0.6045.159 (Official Build) (64-bit)

**Comment by coddingtonbear** [comment]
Thanks for the bug report!  I found the bug and have uploaded a new version of the package to the Chrome App Store for review.  See #163

**Reply by theotheo** [reply]
Thank you!
